libraries/AP_Motors/AP_Motors6DOF.cpp

```diff
@@ -283,9 +289,9 @@ float AP_Motors6DOF::get_current_limit_max_throttle()
 // ToDo calculate headroom for rpy to be added for stabilization during full throttle/forward/lateral commands
 void AP_Motors6DOF::output_armed_stabilizing()
 {
-    if ((sub_frame_t)_last_frame_class == SUB_FRAME_VECTORED) {
+    if ((sub_frame_t)_active_frame_class == SUB_FRAME_VECTORED) {
```

Comment:
Could use `AP_Enum<sub_frame_t> _active_frame_class;`, but I'm pretty sure you've done enough cleanup on this PR :-)